pkg/ui/src/Slideover/Slideover.svelte

```diff
@@ -28,3 +27,20 @@
 		<slot />
 	</div>
 </div>
+
+<style>
+	@keyframes slider {
+		from {
+			transform: translateX(100%);
+			opacity: 0;
+		}
+		to {
+			transform: translateX(0px);
+			opacity: 1;
+		}
+	}
+
+	.animate-unknown {
+		animation: slider 300ms ease-in-out;
```

Comment:
Much faster, excellent! I think it should also animate on its way out. Maybe `slider` could become `slider-in` and `slider-out`.

Reply:
I implemented the easiest way to do it, because it can't be achieved with css alone as far as I know (js must be used to toggle a different class on close).